packages/combobox/test/combobox-a11y.test.ts

```diff
@@ -354,4 +354,31 @@ describe('Combobox accessibility', () => {
 
         expect(el.open).to.be.false;
     });
+    it('manages keyboard navigation and selection', async () => {
```

Comment:
Can we include testing for numeric values, too?

Reply:
Sure! 😄  More than what I already added in `packages/combobox/test/combobox.data.test.ts`? 
Or a combination of numeric values + selection testing?

Reply:
Oh, somehow I didn't see that test. I think we just need to press `Enter` on something there to confirm it's fully hit the `getElementById()` path, and then this will be good to go!

Reply:
I combined them into one test since it was 90% the same, just the data was different. 🙈